src/igvfd/searches/configs/file.py

```diff
@@ -8,122 +8,287 @@ def file():
+                'optional': True,
+                'category': 'Processing Details'
+            },
+            'workflows.uniform_pipeline': {
+                'title': 'Uniformly Processed',
+                'description': 'The status of the single cell or Perturb-seq uniform pipeline processing for this file.',
+                'optional': True,
+                'category': 'Processing Details'
             },
             'file_set.file_set_type': {
-                'title': 'File Set Type'
+                'title': 'File Set Type',
+                'description': 'The type of file set the file belongs to.',
+                'optional': True,
+                'category': 'File Set Details'
             },
             'assay_titles': {
-                'title': 'Assay Term Names'
+                'title': 'Assay',
+                'description': 'High-level assay classification based on OBI assay slim terms.',
+                'optional': True,
+                'category': 'File Set Details'
             },
             'preferred_assay_titles': {
-                'title': 'Preferred Assay Titles'
+                'title': 'Preferred Assay Titles',
+                'description': 'Title of assays that generated the measurement set.',
+                'category': 'File Set Details'
             },
             'cell_type_annotation.term_name': {
-                'title': 'Annotated Cell Type'
+                'title': 'Annotated Cell Type',
+                'description': 'The inferred cell type this file is associated with based on single-cell expression profiling.',
+                'optional': True,
+                'category': 'File Set Details'
             },
             'file_set.samples.taxa': {
-                'title': 'Taxa'
-            },
-            'file_set.samples.sample_terms.term_name': {
-                'title': 'Sample'
+                'title': 'Taxa',
+                'description': 'The organism or species associated with the donor.',
+                'category': 'Sample'
             },
             'file_set.samples.classifications': {
-                'title': 'Classification'
+                'title': 'Classification',
+                'description': 'High-level classification of the sample.',
+                'optional': True,
+                'category': 'Sample'
+            },
+            'file_set.samples.sample_terms.term_name': {
+                'title': 'Sample',
+                'description': 'The sample term name associated with the prediction set.',
+                'optional': True,
+                'category': 'Sample'
             },
             'file_set.samples.targeted_sample_term.term_name': {
-                'title': 'Cellular Transformation Target'
+                'title': 'Cellular Transformation Target',
+                'description': 'Ontology term identifying the targeted endpoint sample resulting from differentiation or reprogramming.',
```

Comment:
"biosample name of the endpoint from a differentiation or reprogramming protocol"